src/gba/cart/vfame.c

```diff
@@ -256,6 +272,8 @@ static uint32_t _modifySramAddress(enum GBAVFameCartType type, uint32_t address,
 		return address;
 	} else if (type == VFAME_GEORGE) {
 		return _reorderBits(address, ADDRESS_REORDERING_GEORGE[mode - 1], 16);
+	} else if (type == VFAME_ALTERNATE) {
+		return _reorderBits(address, ADDRESS_REORDERING_ALTERNATE[mode - 1], 16);
 	} else {
 		return _reorderBits(address, ADDRESS_REORDERING[mode - 1], 16);
 	}
```

Comment:
If it's gonna have more than two values with the same semantics, it's probably a good idea to change this block into a LUT of the orders, indexed by the type